src/pages/dar_application/ProgressReportApplication.tsx

```diff
@@ -45,8 +48,9 @@ export const ProgressReportApplication = ({ dar, datasets, readOnlyMode = true }
     const [dataUseTranslations, setDataUseTranslations] = useState<string[]>([]);
     const [selectedDatasets, setSelectedDatasets] = useState<Dataset[]>(datasets);
     const [formValidation, setFormValidation] = useState<FormValidationState>(
-        {darErrors:
-                {gsoAcknowledgement: {}, pubAcknowledgement: {}, dsAcknowledgement: {}}});
+      {darErrors:
+            {gsoAcknowledgement: {}, pubAcknowledgement: {}, dsAcknowledgement: {}}});
+    const eRACommonsDestination = 'progress_report_application/' + dar?.collectionId;
```

Comment:
```suggestion
    const eRACommonsDestination = 'progress_report_application/' + dar.collectionId;
```

`dar.collectionId` should never be null.